vercel/resource_project_environment_variable_test.go

```diff
@@ -214,7 +214,7 @@ resource "vercel_project_environment_variable" "example_git_branch" {
 resource "vercel_project_environment_variable" "example_sensitive" {
 	project_id = vercel_project.example.id
 	%[3]s
-	key        = "foo_sensitive_updated"
+	key        = "foo_sensitive"
```

Comment:
The test was faulty - updating the key forced a replacement, so it wasn't really testing that the other fields can be updated.